docs/release-notes/RELEASE-NOTES-0.21.0.md

```diff
@@ -18,6 +18,7 @@ It:
 * Changes micro block gossip to use Light micro blocks, containing only Tx hashes. In most cases the receiving node
   has already seen all transactions so this saves bandwidth. This bumps the P2P_PROTOCOL_VSN.
 * Fine-tunes the determinism of the computation of the dynamic component of the fee of oracle transactions related to TTL of objects (oracles, queries, responses) on state trees, moving floating point computations to integer based ones. This impacts consensus.
+* Fixes coinbase instruction in aevm. This impacts consensus.
```

Comment:
Wrong file. Note shall be in https://github.com/aeternity/epoch/wiki/Release-notes-next (as linked in release notes for 0.22.0)

Reply:
Ah yes, this was in a PR for 0.21... first.
Do we have a new process for release notes where we add them to a file that is not contested?
Perhaps we could add a file per PR: PTxxx-notes.txt and then the content of these files could be merged into the release not upon release?